test/Interop/C/function/Inputs/emit-called-function.h

```diff
@@ -0,0 +1,23 @@
+#ifdef __cplusplus
+#define INLINE inline
+#else
+// When compiling as C, make the functions `static inline`. This is the flavor
+// of inline functions for which we require the behavior checked by this test.
+// Non-static C inline functions don't require Swift to emit LLVM IR for them
+// because the idea is that there will we one `.c` file that declares them
+// `extern inline`, causing an out-of-line definition to be emitted to the
+// corresponding .o file.
+#define INLINE static inline
+#endif
+
+INLINE int notCalled() {
+  return 42;
+}
+
+INLINE int calledTransitively() {
+  return 42;
+}
+
+INLINE int calledFromSwift() {
+  return calledTransitively();
```

Comment:
Since there is non-trivial logic in `ClangDeclRefFinder` that distinguishes toplevel inline functions and other functions, WDYT about adding another test where we call an inline member function from a toplevel inline function?

Reply:
Done.